src/Language/PureScript/Docs/Render.hs

```diff
@@ -24,6 +24,15 @@ import qualified Language.PureScript.Environment as P
 import qualified Language.PureScript.Names as P
 import qualified Language.PureScript.Types as P
 
+renderKindSig :: Text -> Text -> Type' -> RenderedCode
+renderKindSig title keywordTxt ty =
+  mintersperse sp
+      [ keyword keywordTxt
+      , ident' title
```

Comment:
This is a temporary solution, but right now this is problematic. Using `ident`' will link to the data constructor that has the same name as the title. In some cases (e.g. `class`), this won't exist and will produce a 404 if clicked.